### DISCUSSION TITLE
is Orama's multiple indexes search supported?

### DISCUSSION THREAD

**Discussion by marcello-stackzero** [fuma-nama/fumadocs#1168] [Q&A]
Hey! I'd like to use Orama's [multiple indexes search](https://docs.orama.com/cloud/performing-search/multi-index-search). 

The main reason I need this is that I want to combine in one single dialog results from 2 indexes: 
- .mdx content indexed from my docs
- JSON data indexed from another source (external)

When I follow the Fumadocs guide for setting up Orama search AND initialize the Orama client as in the Orama docs, I get an error. 

For example, if I do this:

```tsx
'use client';
 
import { OramaClient } from '@oramacloud/client';
import type { SharedProps } from 'fumadocs-ui/components/dialog/search';
import SearchDialog from 'fumadocs-ui/components/dialog/search-orama';
 
// Omitting  api keys 
const client = new OramaClient({
  mergeResults: false
  indexes:[
        {api_key: "<Your Orama Cloud API Key index 1>", endpoint:"<Your Orama Cloud Endpoint index 1>"},
        {api_key: "<Your Orama Cloud API Key index 2>", endpoint:"<Your Orama Cloud Endpoint index 2>"},
      ]
});

 
export default function CustomSearchDialog(props: SharedProps) {
  return <SearchDialog {...props} client={client} showOrama />;
}
```

Is this feature supported? If not, any plans to support it?

Thanks!

**Comment by fuma-nama** [comment]
Group by isn't supported with multi index, and it is still experimental. I will recommend to make your own dialog, or using their official SDK instead of the current one if you need multi index functionality
